lib/android/app/src/main/java/com/reactnativenavigation/react/NavigationModule.java

```diff
@@ -223,7 +223,8 @@ private Options parse(@Nullable ReadableMap mergeOptions) {
 
     protected void handle(Runnable task) {
         UiThread.post(() -> {
-            if (getCurrentActivity() != null && !activity().isFinishing()) {
```

Comment:
I don't understand how it's a different activity, `activity()` returns `getCurrentActivity()`. wdyt?